svf-llvm/include/SVF-LLVM/SVFIRBuilder.h

```diff
@@ -60,6 +60,7 @@ class SVFIRBuilder: public llvm::InstVisitor<SVFIRBuilder>
     /// Destructor
     virtual ~SVFIRBuilder()
     {
+        ExtAPI::destory();
```

Comment:
You assume we don't use ExtAPI after constructing SVFIR? If so, we will need to add an assertion to make sure ExtAPI is only available before SVFIR is constructed.